javascript/code-challenges/tree/README.md

```diff
@@ -20,3 +56,4 @@ I took the approach of creating an add method with and O(h) for time and O(1) fo
 `preOrder()`
 
 
+
```

Comment:
Your code looks really great! Once you have some tests added, I will regrade for you!